src/default_config.toml

```diff
@@ -2,6 +2,14 @@
 # It is possible to override settings with an equivalent file at:
 # `~/.config/gitu/config.toml`
 
+[general]
+
+# enabled can be either of:
+# - true
+# - false
+
```

Comment:
```suggestion
```

I think the option explains itself fairly well!

Reply:
Bah, since this is just a small thing, I'll go ahead and merge it and change it in master. Hope you don't mind! @jondysinger
:rocket: